types/file-saver-es/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for file-saver-es 2.0
+// Project: https://github.com/eligrey/FileSaver.js/
+// Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
+//                 Daniel Roth <https://github.com/DaIgeb>
+//                 Chris Barr <https://github.com/chrismbarr>
+//                 HitkoDev <https://github.com/HitkoDev>
+//                 JounQin <https://github.com/JounQin>
+//                 BendingBender <https://github.com/bendingbender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace saveAs;
```

Comment:
Is this really needed? Exposing all symbols into `window` makes tree shaking unavailable.

Reply:
This is not exposing anything, these are just type definitions. The library is exposing everything into window, nothing I can do about it. But I could lie to users by not mentioning it...